content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,245 @@
+   to match an HTTPRoute.
+4. Optionally, the reverse proxy can perform request header and/or path matching based
+   on match rules of the HTTPRoute.
+5. Optionally, the reverse proxy can modify the request, i.e. add/remove headers, based
+   on filter rules of the HTTPRoute.
+6. Lastly, the reverse proxy forwards the request to one or more objects, i.e. Service,
+   based on rules of the HTTPRoute.
+
+## Conformance
+
+Gateway API covers a broad set of features and is widely implemented. This combination requires
+clear conformance definitions and tests to ensure that the API provides a consistent experience
+wherever it is used.
+
+When considering Gateway API conformance, it's important to understand these concepts:
+
+1. __Release Channels:__ Indicate the stability of a field or resource. The "standard" channel of
+   the API includes fields and resources that have graduated to "beta". The "experimental" channel
+   of the API includes everything in the "standard" channel, along with experimental fields and
+   resources that may still be changed in breaking ways or removed altogether. For more information
+   on this concept, refer to the [versioning][versioning] documentation.
+2. __Support Levels:__ Some implementations of the API may not support every feature. To address this,
+   the API defines _Core_, _Extended_, and _Implementation-specific_ support levels for each feature:
+
+   * __Core__ feature is portable across ALL implementations and is validated by conformance tests. A Core
+     feature is part of the API types schema.
+   * __Extended__ feature is portable but not universally supported across implementations. Implementations
+     that support the feature will have the same behavior and semantics. It is expected that some number of
+     roadmap features will eventually migrate into the _Core_. An _Extended_ feature is part of the API types
+     schema.
+   * __Implementation-specific__ feature is not portable across implementations and is vendor-specific.
+     An _Implementation-specific_ feature will not have API types and schema, except via generic extension
+     points. The implementation is responsible for providing conformance tests to validate these features.
+
+1. __Conformance Tests:__ Gateway API includes a set of conformance tests to validate _Core_ and _Extended_
+   features.
+
+## Getting Started
+
+Instead of Gateway API resources being natively implemented by Kubernetes, the specifications
+are defined as [Custom Resources][crds] supported by a wide range of [implementations][implementations].
+You can [install][install] the Gateway API CRDs manually, but this step is typically accomplished by
+your selected implementation. Use one of the following tabs to install a Gateway API implementation:
+
+{{< tabs name="service_tabs" >}}
+{{% tab name="Default" %}}
+Select one of the tabs.
+{{% /tab %}}
+
+{{% tab name="Envoy Gateway" %}}
```

Comment:
Only one implementation?

Reply:
We have a handful of conformant implementations. I wanted to see what @robscott thought about adding tabs for conformant implementations to this doc and I used Envoy Gateway as an example. I'm not sure if we want this level of detail but having a reference to conformant implementations may incentivize vendors.

xref: https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports

Reply:
We've intentionally avoided considering any Gateway API implementation a reference one, and in this case I'm concerned that these docs would get stale quickly. Instead I'd recommend linking out to the [Gateway API getting started page](https://gateway-api.sigs.k8s.io/guides/). Although that page could use some improvements, it's probably better to keep that kind of content on Gateway API docs where it will be easier for us to keep it up-to-date. For example, we already manually update that page as part of every release.

Reply:
Yeah, I agree that it's probably better to call out to the Getting Started guide on the Gateway API site - this will be super hard to keep up to date.

Reply:
There is currently already precedent in the Kubernetes docs of adding tabs for implementations
https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
would be great if Gateway API can follow this template

It might be best if the tabs link to the Implementations Getting Started page (as suggested by @kflynn )

Reply:
https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer is there because of a bad design choice historically. Let's not use that as a precedent.

Reply:
We have a lot of things to work out here to ensure that we're not picking winners; I don't think that right now is the best time for that, so I'm -1 on having tabs in here _for now_.

So, I recommend we keep this page minimal, and then come back to this after Kubecon. I don't think this one will languish, as we all have incentive to ensure that conformant implementations are properly promoted.

Reply:
Based on the majority of the feedback, I have simplified this section. @arkodg feel free to create a Gateway API issue if you would like to discuss this further.